action.yml

```diff
@@ -15,12 +15,10 @@ runs:
         for dir in $(find . \( -name vendor -o -name '[._].*' -o -name node_modules \) -prune -o -name go.mod -print | sed 's:/go.mod$::'); do
           echo "#=#=#=# $dir #=#=#=#"
           pushd $dir > /dev/null
-          set +e -x
+          s=0
           ( 
```

Comment:
note: you can also say:

```
if ! (
${{ inputs.run }}
); then
status = ...
```

I think?

Reply:
Isn't it great that there are 20 different ways to do the easiest things in bash?

Reply:
That works, but we'd lose the exit status.

Reply:
Ah, I see. Yeah, that makes sense.

>  Isn't it great that there are 20 different ways to do the easiest things in bash?

Hm. That's _way_ too few. We should switch everything to Perl.